go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/gosuri/uilive v0.0.4
 	github.com/jmoiron/sqlx v1.2.0
 	github.com/lib/pq v1.8.0
-	github.com/mattermost/mattermost-operator v1.11.1
+	github.com/mattermost/mattermost-operator v1.11.2-0.20201216193450-e467d2e54786
```

Comment:
Note: It will be updated after https://github.com/mattermost/mattermost-operator/pull/206 is merged and new Operator is released.